packages/currencies/src/data/fiat.js

```diff
@@ -1,39 +1,13 @@
 //@flow
-import type { Unit } from "../types";
+import type { FiatUnit } from "../types";
 // inspired by https://github.com/smirzaei/currency-formatter/blob/master/currencies.json
-const units: { [key: string]: Unit } = {
+const units: { [key: string]: Object } = {
```

Comment:
Can't we use `FiatUnit` instead of `Object`?

Reply:
only if we put the `ticker` inline. maybe we can